wled00/const.h

```diff
@@ -578,8 +578,14 @@
 #else
   #define DEFAULT_LED_PIN 16   // aligns with GPIO2 (D4) on Wemos D1 mini32 compatible boards (if it is unusable it will be reassigned in WS2812FX::finalizeInit())
 #endif
-#define DEFAULT_LED_TYPE TYPE_WS2812_RGB
+
+#ifndef DEFAULT_LED_TYPE
+  #define DEFAULT_LED_TYPE TYPE_WS2812_RGB
+#endif
+
+#ifndef DEFAULT_LED_COUNT
 #define DEFAULT_LED_COUNT 30
```

Comment:
can you fix indentation please?

Reply:
yes, sorry!

Reply:
just a little hint, please: should I commit and PR again or there is a easyest way?

Reply:
just edit your branch, then commit, it will show up here